app/models/app_remote_kill.rb

```diff
@@ -12,14 +12,28 @@ class AppRemoteKill < ActiveRecord::Base
   validate :fixed_version_must_be_greater_than_first_bad_version
   validate :must_not_overlap_existing
 
-  scope :affecting, ->(app_build) do
+  scope :affecting, ->(app_build, override: false, overridden_at: nil) do
     where(
       arel_table[:app_id].eq(app_build.app_id)
       .and(arel_table[:first_bad_version].lteq(app_build.version))
       .and(arel_fixed_version_is_null_or_greater_than(app_build.version))
+      .and(
+        Arel::Nodes::Grouping.new(
+          arel_is_false(override)
+          .or(arel_table[:updated_at].gt(overridden_at))
+        )
+      )
     )
   end
 
+  class << self
+    private
+
+    def arel_is_false(value_or_arel)
+      Arel::Nodes::Grouping.new(Arel::Nodes::False.new).eq(value_or_arel)
```

Comment:
This turns out to be kind of annoying. Arel::Nodes::False doesn't mix in predications, and therefor can't be chained off of with `eq`, so I do this teensy hack and wrap it in a group node (parens) which does.

This way we get the automatic casting behavior of eq, which allows us to get a ruby value object or an arel node and work fine regardless.